frontend/nuxt.config.ts

```diff
@@ -191,10 +190,14 @@ export default defineNuxtConfig({
   },
   plausible: {
     enabled: !isTest,
+    // In dev, the events are not sent anywhere, only logged to the console.
+    // In production builds, ignore events from staging.
+    ignoredHostnames: !isProductionBuild
+      ? ["localhost"]
+      : ["staging.openverse.org"],
```

Comment:
@obulat Since we remove "isProductionBuild" in #4705, should we use it here? I tried to ask this in my earlier comment but wouldn't `["localhost", "staging.openverse.org"]` be a safe value in _all_ environments, regardless of the build type?

Reply:
I added the env variable to enable logging the ignored events when localhost is ignored.